src/polymer/polymer-element.ts

```diff
@@ -221,14 +221,15 @@ function _getFlattenedAndResolvedBehaviors(
     //     array, which should be the most recently defined one.
     const foundBehavior = document.getOnlyAtId('behavior', behavior.name);
     if (!foundBehavior) {
-      throw new WarningCarryingException({
+      document.warnings.push({
```

Comment:
Hm, good point. I think the solution there is for us to also call something like getFlattenedAndResolvedBehaviors as part of resolving Behaviors.